views/layout.hbs

```diff
@@ -1,26 +1,36 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <style>
+    </style>
     <title>{{title}}</title>
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
     <link rel='stylesheet' href='/stylesheets/style.css' />
   </head>
   <body>
+
     <nav class="navbar navbar-expand-lg navbar-light bg-light" id="aceBookNavBar">
       <a class="navbar-brand" href="/">Acebook</a>
       <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
         <span class="navbar-toggler-icon"></span>
       </button>
-      <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+
+    <div class="collapse navbar-collapse" id="navbarNavAltMarkup"></div>
```

Comment:
Would you be ok to clean these up so that it is easier to read? :) When you open a have a child contained in an element such as a div try to tab one further eg:
<div>
    <div>
        <form>
        </form>
    </div>
</div>

Reply:
not sure you can see my eg so let me know if you need me to clarify